gradle/wrapper/gradle-wrapper.properties

```
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-all.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```
We should do 4.10.2. IIRC 4.10 had some pretty bad bugs